src/Misc/externals.sh

```diff
@@ -18,6 +18,8 @@ get_abs_path() {
 DOWNLOAD_DIR="$(get_abs_path "$(dirname $0)/../../_downloads")/$PACKAGERUNTIME/netcore2x"
 if [[ "$LAYOUT_DIR" == "" ]]; then
     LAYOUT_DIR=$(get_abs_path "$(dirname $0)/../../_layout/$PACKAGERUNTIME")
+else
```

Comment:
This LAYOUT_DIR in externals was not getting passed in correctly for me. I am not sure why. Changing the parameter to just be a relative folder worked a lot better.